include/s1ap/s1ap_macros.h

```diff
@@ -0,0 +1,36 @@
+/*
+* Copyright (c) 2003-2020, Great Software Laboratory Pvt. Ltd.
+*
+* SPDX-License-Identifier: Apache-2.0
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*  http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*
+*/
+
+#ifndef __S1AP_MACROS_H_
+#define __S1AP_MACROS_H_
+
+#include <stdbool.h>
+
+
+enum e_rab_struct_val
+{
+  RAB_ID = 1,
```

Comment:
These are default values assigned for time being. Ideally, we should replace with the correct values from the transaction.
Please name defines properly to represent default values or actual meaning. E.g. PRIOLEVEL is not the right name. What priority does number 15 represents should be mentioned. Give the spec references wherever possible.

Reply:
@kanchankanchan  can you respond to comment ?